backend/slickMysql/src/main/scala/io.buildo.toctoc/slick/authentication/login/MySqlSlickLoginAuthenticationDomain.scala

```diff
@@ -0,0 +1,60 @@
+package io.buildo.toctoc
+package slick
+package authentication
+package login
+
+import core.authentication._
+import core.authentication.TokenBasedAuthentication._
+
+import cats.data.EitherT
+import cats.implicits._
+import _root_.slick.jdbc.MySQLProfile.api._
+import _root_.slick.jdbc.JdbcBackend.Database
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class MySqlSlickLoginAuthenticationDomain(db: Database)(implicit ec: ExecutionContext)
+  extends LoginAuthenticationDomain
+  with BCryptHashing {
+
+  class LoginTable(tag: Tag) extends Table[(Int, String, String, String)](tag, "login_auth_domain") {
+    def id = column[Int]("id", O.PrimaryKey, O.AutoInc)
+    def ref = column[String]("ref")
+    def username = column[String]("username", O.Length(255))
```

Comment:
You're right: I should set 45 instead of 225.
This is a slick-mysql constraint: if I use the field in a key or say that it must be unique, it must have a length declaration...